src/rttable.c

```diff
@@ -712,40 +702,40 @@ int internUpdateKernelRoute(struct RouteTable *route, int activate) {
 *   to the log.
 */
 void logRouteTable(const char *header) {
-        struct RouteTable*  croute = routing_table;
-        unsigned            rcount = 0;
-    
-        my_log(LOG_DEBUG, 0, "");
-        my_log(LOG_DEBUG, 0, "Current routing table (%s):", header);
-        my_log(LOG_DEBUG, 0, "-----------------------------------------------------");
-        if(croute==NULL) {
-            my_log(LOG_DEBUG, 0, "No routes in table...");
-        } else {
-            do {
-                char st = 'I';
-                char src[MAX_ORIGINS * 30 + 1];
-                src[0] = '\0';
-                
-                for (int i = 0; i < MAX_ORIGINS; i++) {
-                    if (croute->originAddrs[i] == 0) {
-                        continue;
-                    }
-                    st = 'A';
-                    sprintf(src + strlen(src), "Src%d: %s, ", i, inetFmt(croute->originAddrs[i], s1));
+    struct RouteTable*  croute = routing_table;
+    unsigned            rcount = 0;
+
+    my_log(LOG_DEBUG, 0, "");
+    my_log(LOG_DEBUG, 0, "Current routing table (%s):", header);
+    my_log(LOG_DEBUG, 0, "-----------------------------------------------------");
+    if(croute==NULL) {
+        my_log(LOG_DEBUG, 0, "No routes in table...");
+    } else {
+        do {
+            char st = 'I';
+            char src[MAX_ORIGINS * 30 + 1];
+            src[0] = '\0';
+
+            for (int i = 0; i < MAX_ORIGINS; i++) {
+                if (croute->originAddrs[i] == 0) {
+                    continue;
                 }
-                
-                my_log(LOG_DEBUG, 0, "#%d: %sDst: %s, Age:%d, St: %c, OutVifs: 0x%08x",
-                    rcount, src, inetFmt(croute->group, s2),
-                    croute->ageValue, st,
-                    croute->vifBits);
-                  
-                croute = croute->nextroute; 
-        
-                rcount++;
-            } while ( croute != NULL );
-        }
-    
-        my_log(LOG_DEBUG, 0, "-----------------------------------------------------");
+                st = 'A';
+                sprintf(src + strlen(src), "Src%d: %s, ", i, inetFmt(croute->originAddrs[i], s1));
+            }
+
+            my_log(LOG_DEBUG, 0, "#%d: %sDst: %s, Age:%d, St: %c, OutVifs: 0x%08x",
+                rcount, src, inetFmt(croute->group, s2),
+                croute->ageValue, st,
+                croute->vifBits);
+
+            croute = croute->nextroute; 
+
+            rcount++;
+        } while ( croute != NULL );
+    }
+
+    my_log(LOG_DEBUG, 0, "-----------------------------------------------------");
```

Comment:
Hi! I think that this commit 3fb62e50fc9b953f73de8eaaaa12ed464c215327 does not clean up code and make it consistent. E.g. in this line you do not use spaces around brackets...